### PR TITLE
fix(create-mud): set store address in PostDeploy script

### PR DESCRIPTION
**Pull request by 0xrhsmt**
fixes https://github.com/latticexyz/mud/issues/1719

### CODE REVIEW

**Review by holic — APPROVED** [review]

**Review by holic — APPROVED** [review]
thank you!

**Comment by alvrs** [comment]
I think we might have to import `StoreSwitch` first

```solidity
import { StoreSwitch } from "@latticexyz/store/src/StoreSwitch.sol";
```

File: examples/minimal/packages/contracts/script/PostDeploy.s.sol [hunk 1]
```diff
@@ -12,6 +12,9 @@ import { ChatNamespacedSystem } from "../src/systems/ChatNamespacedSystem.sol";
 
 contract PostDeploy is Script {
   function run(address worldAddress) external {
+    // Specify a store so that you can use tables directly in PostDeploy
+    StoreSwitch.setStoreAddress(worldAddress);
```
Comment (holic) [inline examples/minimal/packages/contracts/script/PostDeploy.s.sol:16]: looks like this is missing an import of `StoreSwitch`

Comment (holic) [inline examples/minimal/packages/contracts/script/PostDeploy.s.sol:16]: looks like this is missing an import of `StoreSwitch`

**Review by holic — CHANGES_REQUESTED** [review]

File: examples/minimal/packages/contracts/script/PostDeploy.s.sol (hunk 1 above)
Comment (0xrhsmt) [reply examples/minimal/packages/contracts/script/PostDeploy.s.sol:16]: Sorry, fixed it. 🙏

Comment (0xrhsmt) [reply examples/minimal/packages/contracts/script/PostDeploy.s.sol:17]: Sorry, fixed it. 🙏

**Pull request merged**